extras/create_snapshots_reports_scorecard.py

```diff
@@ -1021,6 +1012,39 @@ def main():
                 else:
                     logging.error(i)
 
+        if not args["--no-snapshots"]:
+            logging.info("Snapshot performance:")
+            durations = [x[1] for x in snapshot_durations]
+            max = numpy.max(durations)
+            logging.info("  Maximum snapshot: %.1f seconds (%.1f minutes)", max, max / 60)
```

Comment:
I'm just not a fan of "Maximum" in the context of runtime. This also matches the language of the 10 longest output.

```suggestion
            logging.info("  Longest snapshot: %.1f seconds (%.1f minutes)", max, max / 60)
```